qemu/tests/cfg/timedrift_adjust_time.cfg

```diff
@@ -1,5 +1,5 @@
 - timedrift_adjust_time:
-    no RHEL.3,4,5 WinXp Win2000 Win2003 WinVista
+    no RHEL.3, RHEL.4, RHEL.5, WinXp, Win2000, Win2003, WinVista
```

Comment:
Don't need to add `no RHEL.3, RHEL.4, RHEL.5`, avocado it self does no support them anymore.

Reply:
Thanks, will update it.